### PR TITLE
Bug 1574836 Edited AWS info to be more specific about labeling

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1574836

Will verify in the BZ.

### CODE REVIEW

**Comment by bfallonf** [comment]
Verified from QA in the BZ.

@openshift/team-documentation PTAL

Context: It was vague on the level of tagging that needed to happen on an AWS install. I tried to make it more specific around "relevant hosts/resources" instead of just "hosts/resources", etc.

File: install_config/configuring_aws.adoc [hunk 1]
```diff
@@ -12,10 +12,11 @@ toc::[]
...
 == Overview
 {product-title} can be configured to access an
-link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html[AWS EC2 infrastructure], including
-xref:../install_config/persistent_storage/persistent_storage_aws.adoc#install-config-persistent-storage-persistent-storage-aws[using AWS
-volumes as persistent storage] for application data. After AWS is configured
-properly, some additional configurations will need to be completed on the
+link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html[AWS EC2
+infrastructure], including
+xref:../install_config/persistent_storage/persistent_storage_aws.adoc#install-config-persistent-storage-persistent-storage-aws[using
+AWS volumes as persistent storage] for application data. After AWS is configured
```
Comment (kalexand-rh) [inline install_config/configuring_aws.adoc]: Maybe s/After AWS is configured properly, some additional configurations must be completed on the {product-title} hosts./After you configure AWS, you must modify the configuration of the {product-title} hosts.

File: install_config/configuring_aws.adoc [hunk 2]
```diff
@@ -310,18 +310,26 @@ There are four types of resources that need to be tagged:
 [[aws-tagging-an-existing-cluster]]
 === Tagging an Existing Cluster
 
-A cluster uses the value of the `kubernetes.io/cluster/<name>,Value=<clusterid>` tag to determine which
-resources belong to the cluster. Therefore, you must tag all resources with the
-key `kubernetes.io/cluster/<name>,Value=<clusterid>` and have the same value for that key.
+A cluster uses the value of the `kubernetes.io/cluster/<name>,Value=<clusterid>`
+tag to determine which resources belong to the AWS cluster. Therefore, tag all
+relevant resources with the `kubernetes.io/cluster/<name>,Value=<clusterid>`
+key, and have the same value for that key.
```
Comment (kalexand-rh) [inline install_config/configuring_aws.adoc]: I think the way that we're referring to the key is a little confusing. 
Can you add a note here that explains the `name` and `clusterid` parameter values? It sounds like `clusterid` needs to be some constant, arbitrary value, but IDK what the `name` should be.

File: install_config/configuring_aws.adoc [hunk 3]
```diff
@@ -310,18 +310,26 @@ There are four types of resources that need to be tagged:
...
+. Tag the relevant security groups to be used in the AWS instance with `kubernetes.io/cluster/<name>,Value=<clusterid>` and the same value used for the instances.
+. Tag the relevant load balancers to be used in the AWS instance with `kubernetes.io/cluster/<name>,Value=<clusterid>` and the same value used for the instances.
 . Tag all EBS volumes with `kubernetes.io/cluster/<name>,Value=<clusterid>` and the same value used for the instances. The EBS Volumes that need to be tagged can found with:
 +
 [source,bash]
 ----
 $ oc get pv -o json|jq '.items[].spec.awsElasticBlockStore.volumeID'
 ----
++
+[NOTE]
```
Comment (kalexand-rh) [inline install_config/configuring_aws.adoc]: I think this should go with the step to tag the security groups.

File: install_config/configuring_aws.adoc [hunk 4]
```diff
@@ -310,18 +310,26 @@ There are four types of resources that need to be tagged:
...
-key `kubernetes.io/cluster/<name>,Value=<clusterid>` and have the same value for that key.
+A cluster uses the value of the `kubernetes.io/cluster/<name>,Value=<clusterid>`
+tag to determine which resources belong to the AWS cluster. Therefore, tag all
+relevant resources with the `kubernetes.io/cluster/<name>,Value=<clusterid>`
+key, and have the same value for that key.
 
-. Tag all instances with `kubernetes.io/cluster/<name>,Value=<clusterid>` and a value to be used as the cluster ID.
-. Tag any security groups with `kubernetes.io/cluster/<name>,Value=<clusterid>` and the same value used for the instances.
-. Tag any load balancers with `kubernetes.io/cluster/<name>,Value=<clusterid>` and the same value used for the instances.
+. Tag all hosts with `kubernetes.io/cluster/<name>,Value=<clusterid>` and a value to be used as the cluster ID.
```
Comment (kalexand-rh) [inline install_config/configuring_aws.adoc]: If you add a note before these steps, I'd remove "and a value to be used as the cluster ID" and similar from these three steps. (I think you're saying that the clusterID value is constant in all three, but I'm not sure.)

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of suggestions and a request.

File: install_config/configuring_aws.adoc (hunk 4 above)
Comment (bfallonf) [reply install_config/configuring_aws.adoc]: I see what you mean. I've updated the points to take away the fluff.

**Comment by bfallonf** [comment]
Thanks @kalexand-rh . I've taken in the suggestions. I'll merge.

**Comment by kalexand-rh** [comment]
It looks beautiful, @bfallonf!

**Pull request merged**